pulp_ostree/app/serializers.py

```diff
@@ -95,9 +95,14 @@ class OstreeRefSerializer(platform.SingleArtifactContentSerializer):
         queryset=models.OstreeCommit.objects.all(),
     )
     name = serializers.CharField()
+    commit_checksum = serializers.CharField(source="commit.checksum")
```

Comment:
Should we mark this read_only?

Reply:
I am not sure if this is necessary. We do not use the serializer for creating new content, just for viewing the existing content.

Reply:
We might in the future, and this is the first point i see it breaking...